deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -63,6 +63,10 @@ def __init__(
         self.train_properties = defaultdict(list)
         self.test_properties = defaultdict(list)
 
+    def initialize_run(self, context: Context):
```

Comment:
Is this added in other places where it's needed?

Reply:
not yet, PR PR